examples/core/core_custom_frame_control.c

```diff
@@ -27,12 +27,20 @@
 ********************************************************************************************/
 
 #include "raylib.h"
+#include "config.h"
+#include "assert.h"
 
 //------------------------------------------------------------------------------------
 // Program main entry point
 //------------------------------------------------------------------------------------
 int main(void)
 {
+    #if !defined(SUPPORT_CUSTOM_FRAME_CONTROL)
+    // This examaple requires the SUPPORT_CUSTOM_FRAME_CONTROL feature.
+    // Please edit your raylib config.h and recompile raylib.
+    assert(false);
+    #endif
```

Comment:
Why is assert needed? would not a printf and return -1 here be the same?
assert is not use any other raylib code.

Reply:
Inside the library assert might be to harsh, 
but considering this example application can not function without that define, I would use an assert.

But whatever the codebase follows is fine for me